instrumentation-genai/opentelemetry-instrumentation-openai-v2/src/opentelemetry/instrumentation/openai_v2/package.py

```diff
@@ -14,3 +14,5 @@
 
 
 _instruments = ("openai >= 1.26.0",)
+
+_supports_metrics = True
```

Comment:
what does it do? (for my own education)

Reply:
It controls the print of `yes` in the `metrics support` in the table :)